app/views/shared/_reference_plans_list.html.erb

```diff
@@ -29,7 +29,8 @@
 
   <div class="health">
     <ul class="nav nav-tabs row">
-	<li role="presentation" class="col-xs-4" data-offering-id="<%= f.object.id %>">
+      <% if individual_market_is_enabled? %>
```

Comment:
this method doesn't have anything to do with sole source.

I'd much prefer you create a settings entry and a new helper method and use that.

ex ) def sole_source_only?